lib/commands/debug/helper.js

```diff
@@ -0,0 +1,143 @@
+        + `"--skillStreamHandlerClass ${skillClass}"`,
+        ext: 'java'
+    });
+}
+
+function invokeNodeDebugger(token, skillId, codeFolder, skillFilePath, handlerName) {
+    nodemon({
+        script: `./${codeFolder}/node_modules/ask-sdk-local-debug/dist/LocalDebuggerInvoker.js`,
+        args: ['--accessToken', `"${token}"`, '--skillId', skillId,
+            '--handlerName', handlerName, '--skillEntryFile', path.join(process.cwd(), skillFilePath)]
+    });
+}
+
+function invokeAttachNodeDebugger(debugPort, token, skillId, codeFolder, skillFilePath, handlerName) {
+    nodemon({
+        execMap: {
+            js: `node --inspect-brk=${debugPort}`,
+        },
+        script: `./${codeFolder}/node_modules/ask-sdk-local-debug/dist/LocalDebuggerInvoker.js`,
+        args: ['--accessToken', `"${token}"`, '--skillId', skillId,
+            '--handlerName', handlerName, '--skillEntryFile', path.join(process.cwd(), skillFilePath)]
+    });
+}
+
+function invokePythonDebugger(token, localDebuggerPath, skillId, handlerName, skillFilePath) {
+    nodemon({
+        execMap: {
+            py: 'python3'
+        },
+        script: localDebuggerPath,
+        args: ['--accessToken', `"${token}"`, '--skillId', skillId,
+            '--skillHandler', handlerName, '--skillFilePath', skillFilePath],
+        ext: 'py,json,txt'
+    });
+}
+
+function invokeAttachPythonDebugger(debugPort, token, localDebuggerPath, skillId, handlerName, skillFilePath) {
+    execSync('python3 -m pip install debugpy', { stdio: 'inherit' });
+    nodemon({
+        execMap: {
+            py: `python3 -m debugpy --listen ${debugPort} --wait-for-client`
+        },
+        script: localDebuggerPath,
+        args: ['--accessToken', `"${token}"`, '--skillId', skillId,
+            '--skillHandler', handlerName, '--skillFilePath', skillFilePath],
+        ext: 'py,json,txt'
+    });
+}
+
+function getSkillInvocationInfo(handler, runtime, isHosted) {
+    if (runtime.includes('node') || runtime.includes('python')) {
+        if (isHosted) {
+            if (runtime.includes('node')) {
+                return { handlerName: 'handler', skillFileName: 'index' };
+            }
+            if (runtime.includes('python')) {
+                return { handlerName: 'lambda_handler', skillFileName: 'lambda_function' };
+            }
+        }
+        const handlerInfo = handler.split('.');
+        const handlerName = handlerInfo.pop();
+        const skillFileName = handlerInfo.join(path.delimiter);
+        return { handlerName, skillFileName };
+    }
+    if (runtime.includes('java')) {
+        return { handlerName: handler };
+    }
+}
+
+function initiateDebugging(runtime, skillInvocationInfo, skillCodeFolderName, waitForAttach, debugPort, token, skillId) {
+    if (runtime.includes('python')) {
+        const localDebuggerPath = getPythonDebugAdapterPath();
+        if (localDebuggerPath === undefined) {
+            throw new CliError('Install ask-sdk-local-debug');
```

Comment:
Great customer obsession!

Reply:
I have given the link to instructions page. Let me know if that works.

Reply:
@daltonhuynh let me know if this works for you?

Reply:
I like offering the command, since it gives developers a solution they can use immediately. Maybe we can go with that and also add a link to the documentation at the end, saying "For more information, visit..."?